### PR TITLE
Ensure DAX cluster endpoints use TLS

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/terraform/checks/resource/aws/example_DAXEndpointTLS/main.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  }
+  tags                 = { test = "Fail" }
+}
+
+resource "aws_dax_cluster" "fail2" {
+  cluster_name         = var.cluster_name
+  iam_role_arn         = var.iam_role_arn
+  parameter_group_name = aws_dax_parameter_group.example.name
+  subnet_group_name    = aws_dax_subnet_group.example.name
+  cluster_endpoint_encryption_type = "NONE"
```
Comment (gruebel) [inline tests/terraform/checks/resource/aws/example_DAXEndpointTLS/main.tf:18]: I think you meant to remove this line 🙂

**Review by gruebel — COMMENTED** [review]
other than that, looks good 👍

File: tests/terraform/checks/resource/aws/example_DAXEndpointTLS/main.tf (hunk 1 above)
Comment (JamesWoolfenden) [reply tests/terraform/checks/resource/aws/example_DAXEndpointTLS/main.tf]: indeed i did

**Review by gruebel — APPROVED** [review]
🍻

**Pull request merged**